templates/home.html

```diff
@@ -10,10 +10,10 @@
       <ol class="breadcrumb my-4">
         <li class="breadcrumb-item active">Boards</li>
       </ol>
-    <table border="1">
-      <thead>
+    <table class="table">
```

Comment:
Element `table` with class `table`... That's either redundant or not very descriptive, as you want classes for specific things like: `boardsTable` or `pipetasTableBlue` and then apply css to them. I know you were following the tutorial, but I thought it was worth it mentioning it.

In CSS, you can apply the style to the element, in this case `table`:

```
table {
    border=1;
}
```
or the class, in this case..... `table` again.....(you see why it is confusing?):

```
.table {
    border=1;
}
```

First one applies to all elements `table`, second one applies to all elements with class="table". 😄